DevCycle/Networking/DevCycleService.swift

```diff
@@ -176,6 +179,11 @@ class DevCycleService: DevCycleServiceProtocol {
                     completion?((nil, nil, error))
                     return
                 }
+                
+                if let urlString = response?.url?.absoluteString {
```

Comment:
Should we return first and log later? You're gonna add a few ms by doing string stuff first no?

Reply:
yea its going to be like a couple of CPU cycles, not a few ms. Also, the callback handler executes an anonymous function that could do a bunch of work b4 the log.